src/installer/corehost/cli/hostpolicy/hostpolicy.cpp

```diff
@@ -448,15 +476,16 @@ SHARED_API int HOSTPOLICY_CALLTYPE corehost_main_with_output_buffer(const int ar
     return rc;
 }
 
-int corehost_libhost_init(const hostpolicy_init_t &hostpolicy_init, const pal::string_t& location, arguments_t& args)
+int corehost_libhost_init(const hostpolicy_init_t &hostpolicy_init, const pal::string_t& location)
```

Comment:
We should rename this as well - `trace_corehost_libhost_init`. It would make it consistent with the new `trace_corehost_init` and it would also nicely document that it doesn't change any state.

Reply:
Also - no reason to return status code - it can't fail.